pkgs/development/python-modules/cppe/default.nix

```diff
@@ -0,0 +1,36 @@
+{ buildPythonPackage, lib, cmake, cppe, eigen
+, python, pybind11, numpy, h5py, numba, scipy, pandas, pytestCheckHook
+}:
+
+buildPythonPackage rec {
+  inherit (cppe) pname version src meta;
+
+  # The python interface requires eigen3, but builds from a checkout in tree.
+  # Using the nixpkgs version instead.
+  postPatch = ''
+    substituteInPlace setup.py \
+      --replace '"external/eigen3",' '"${eigen}/include/eigen3",'
+  '';
+
+  nativeBuildInputs = [ cmake ];
+  dontUseCmakeConfigure = true;
+
+  buildInputs = [ pybind11 eigen ];
```

Comment:
Is a binary from eigen called?

Reply:
No, it just uses its header files. You mean better move it to `nativeBuildInputs`?